src/brevitas/graph/gpfq.py

```diff
@@ -4,199 +4,40 @@
 from copy import deepcopy
 import math
 from typing import List, Optional
+import warnings
```

Comment:
This is ok, but from now on I want to stop using warning and move everything to logging, so we have a bit more control